engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -83,9 +84,17 @@ public void sendEmail(User user, Event event, TemplateInstance subject, Template
         LocalDateTime start = LocalDateTime.now(UTC);
 
         Timer.Sample processedTimer = Timer.start(registry);
+
         EventType eventType = event.getEventType();
-        String bundleName = eventType.getApplication().getBundle().getName();
-        String applicationName = eventType.getApplication().getName();
+        String bundleName = "NA";
+        String applicationName = "NA";
+        if (eventType != null) {
+            bundleName = eventType.getApplication().getBundle().getName();
+            applicationName = eventType.getApplication().getName();
+        } else if (event.getEventWrapper().getEvent() instanceof Action action) {
+            bundleName = action.getBundle();
+            applicationName = action.getApplication();
+        }
```

Comment:
This should always be true, we are not aggregating with cloud events yet - could you move this to the first option?

Reply:
disregard my previous comment.

Reply:
The first option shouldn't be trying to get App and Bundle from eventType ? because we have much more instant notifications than aggregated notifications ?